src/pybind/rbd/rbd.pyx

```diff
@@ -700,71 +700,110 @@ class OSError(Error):
         return (self.__class__, (self.message, self.errno))
 
 class PermissionError(OSError):
-    pass
+    def __init__(self, msg):
```

Comment:
You dropped the ```errno``` keyword argument, so all of these will fail to be constructed in ```make_ex```

Reply:
@iotcg i think what Jason means is that before your change, the exception classes inherited from `OSError` will can constructed using `make_ex()` which is used everywhere in this module. and this function does use the `errno` keyword argument. your change simply removes the inherited constructor. and it almost breaks all the callers of `make_ex()`.